usaspending_api/awards/models.py

```diff
@@ -500,7 +500,7 @@ class Subaward(DataSourceTrackedModel):
     award = models.ForeignKey(Award, models.CASCADE, related_name="subawards")
     recipient = models.ForeignKey(LegalEntity, models.DO_NOTHING)
     submission = models.ForeignKey(SubmissionAttributes, models.CASCADE)
-    cfda = models.ForeignKey(CFDAProgram, models.DO_NOTHING, null=True)
+    cfda = models.ForeignKey(CFDA, models.DO_NOTHING, related_name="related_subawards", null=True)
```

Comment:
What do you think about `related_cfda` or `cfda_subawards` or whatnot here, to make it clear that we're talking about CFDA?